### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by the-og-gear**
[Changelogs]: # (Please make a changelog if you're adding, removing or changing content that'll affect players. This includes, but is not limited to, new features, sprites, sounds; balance changes; map edits and important fixes)
[]: # (See here for how to easily make a changelog: https://github.com/tgstation/tgstation/wiki/Changelogs. An example changelog has been provided below. Please edit or remove)

:cl: MASMC
add: Added a sulpheric acid dispenser
imageadd: Added the icon for the sulpheric acid dispenser
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
you don't need to keep the top part of the description

**Comment by ghost** [comment]
also remember to name your PR so it's easier to find later, not just "updates" that could mean anything

**Comment by the-og-gear** [comment]
Alright. I'll make sure to do that next time.

**Comment by TalkingCactus** [comment]
`sulfuric`

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -190,3 +190,10 @@
 	icon_state = "virus_food"
 	anchored = 1
 	reagent_id = "virusfood"
+
+/obj/structure/reagent_dispensers/sulphericacid
+	name = "sulpheric acid dispenser"
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_dispenser.dm:195]: you misspelled it

**Pull request closed without merging**